### ISSUE TITLE
Use images in place of emojis and wingdings.

### ISSUE THREAD

**Issue by JasonSwindle** [kickscondor/fraidycat#90]
**Desktop (please complete the following information):**
 - OS: Mac
 - Browser: Firefox
 - Version: 73.0.1 (64-bit)

**Describe the bug**
I am seeing a box that says:

```
2B
AB
```

or

```
0lF
7AB
```

at the bottom of every entry.

**Comment by JasonSwindle** [comment]
This looks to be the issue:

https://github.com/kickscondor/fraidycat/blob/master/src/js/view.js#L401-L402

**Comment by kickscondor** [comment]
Yes, this is important to me. Just have been lazy. Thanks for reminding me - I had wanted to add this issue. Jason Swindle!

**Comment by JasonSwindle** [comment]
For the ..., this may work:

```
…
HORIZONTAL ELLIPSIS
Unicode: U+2026, UTF-8: E2 80 A6
HTML Entity (decimal): &#8230
HTML Entity (hex: &#x2026
HTML Entity (named): &hellip
```

**Comment by Quenz** [comment]
I think I'm experiencing the same issue.

 - OS: Mac
 - Browser: Firefox, Brave, Fraidycat app
 - Version: 1.1.2 (Firefox), 1.1.1 (Brave), 1.1.2 (app)

The blue 'expand' button next to someone you follow has some characters that aren't rendering on my system.

Screenshot of characters not rendering in Firefox:

<img width="1378" alt="Screen Shot 2020-03-13 at 10 31 12 AM" src="https://user-images.githubusercontent.com/17078658/76576792-4c7af080-6517-11ea-9b21-b4533d56bd56.png">

**Comment by kickscondor** [comment]
See also #137. Arrows need to be replaced too.

**Comment by kickscondor** [comment]
Ok most of these have been fixed in 3ad33b5.
